src/tests/fetchPrizes.test.js

```diff
@@ -0,0 +1,9 @@
+import fetchPrizeData from '../services/fetchPrizes';
+
+describe('Fetch Data', () => {
+  test('gets the correct data', async () => {
+    const data = await fetchPrizeData();
+    expect(data).toBeInstanceOf(Object);
+    expect(data.id).not.toBeNull();
+  });
+});
```

Comment:
- [ ] Great job having a test file. But you need to add more tests, both unit and integration tests. See screenhot below of what you have to add for your testing to be as per project requirements.
![Screen Shot 2023-02-15 at 5 11 22 AM](https://user-images.githubusercontent.com/62849872/219036728-ad1b78ad-00d3-4fd1-97a1-915eb1848476.png)